.github/workflows/generate_files.yml

```diff
@@ -20,7 +23,13 @@ jobs:
           cargo build --verbose
           cargo hacspec -e v --dir ../coq/src/ --vc-dir ../coq/ --vc-update "*"
           # cargo hacspec -e fst --dir ../fstar/src/ --vc-dir ../fstar/ --vc-update "*"
-      - uses: stefanzweifel/git-auto-commit-action@v4
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
         with:
-          commit_message: "Automated file generation"
-          file_pattern: "coq/src/*.v coq/_vc/*.v"
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: Add generated files
+          title: '[Examples] Add generated files'
+          body: >
+            This PR is auto-generated by
```

Comment:
Let's link to this action (file) instead to make it clear where it's coming from.